### ISSUE TITLE
Repeating errors in log

### ISSUE THREAD

**Issue by para2022** [WWBN/AVideo#8491]
[02-Oct-2023 08:47:57 Europe/London] AVideoLog::ERROR: Time: 2.737s   | Limit: 0.7s | Location: path/to/my/streamer/site/view/modeYoutube.php Line 1584 [getVideosURL_V2::VideoHLS::getSourceFile(video_220601233201_vbf2e)] USER_AGENT=Mozilla/5.0 (Macintosh; Intel Mac OS X 10_10_1) AppleWebKit/600.2.5 (KHTML, like Gecko) Version/8.0.2 Safari/600.2.5 (Amazonbot/0.1; +https://developer.amazon.com/support/amazonbot) backtrace=[{"file":"\/home\/myMySQLUser\/public_html\/myMySQLUser\/objects\/functions.php","line":1584,"function":"TimeLogEnd","args":["getVideosURL_V2::VideoHLS::getSourceFile(video_220601233201_vbf2e)",1584]},{"file":"\/home\/myMySQLUser\/public_html\/myMySQLUser\/objects\/functions.php","line":1416,"function":"getVideosURL_V2","args":["video_220601233201_vbf2e"]},{"file":"\/home\/myMySQLUser\/public_html\/myMySQLUser\/plugin\/VideoResolutionSwitcher\/VideoResolutionSwitcher.php(18) : eval()'d code(1) : eval()'d code","line":99,"function":"getVideosURL","args":["video_220601233201_vbf2e"]},{"file":"\/home\/myMySQLUser\/public_html\/myMySQLUser\/objects\/functions.php","line":1783,"function":"getVRSSources","args":["video_220601233201_vbf2e",true]},{"file":"\/home\/myMySQLUser\/public_html\/myMySQLUser\/view\/modeYoutube.php","line":235,"function":"getSources","args":["video_220601233201_vbf2e",true]}] SCRIPT_NAME: /view/modeYoutube.php

 [02-Oct-2023 08:47:57 Europe/London] AVideoLog::ERROR: Time: 2.741s   | Limit: 0.5s | Location: path/to/my/streamer/site/view/modeYoutube.php Line 245 [modeYoutube.php] USER_AGENT=Mozilla/5.0 (Macintosh; Intel Mac OS X 10_10_1) AppleWebKit/600.2.5 (KHTML, like Gecko) Version/8.0.2 Safari/600.2.5 (Amazonbot/0.1; +https://developer.amazon.com/support/amazonbot) backtrace=[{"file":"\/home\/myMySQLUser\/public_html\/myMySQLUser\/view\/modeYoutube.php","line":245,"function":"TimeLogEnd","args":["modeYoutube.php",245,0.5]}] SCRIPT_NAME: /view/modeYoutube.php

**Comment by DanielnetoDotCom** [comment]
This is not an error; it is only to call your attention to what is consuming high resources from your server.

in your case, I can see the Amazonbot is scanning your site and consuming too much

**Comment by para2022** [comment]
But it is saturating my log. How can i stop this behaviour?

**Comment by DanielnetoDotCom** [comment]
I am surprised that is it taking that long. it maximum may happen only once per video. and will only do it again if you clear the cache

are you using SSD drives?

**Comment by para2022** [comment]
yes ssd drives. My site is super fast with cache off. This started in my logs a few days ago. Never ever seen it before.
